### PR TITLE
OSDOCS#6410: Release notes: user-defined tags for Azure is generally available 

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.14 only

Issue: https://issues.redhat.com/browse/OSDOCS-6410

Link to docs preview:  https://xenolinux.github.io/share-previews/azure-user-tags-ga/release_notes/ocp-4-14-release-notes.html#ocp-4-14-user-defined-tags-azure

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
Screenshot 2023-06-07 at 7 01 47 PM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -51,6 +51,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-installation-and-update"]
 === Installation and update
 
+[id="ocp-4-14-user-defined-tags-azure"]
+==== User-defined tags for Microsoft Azure is now generally available
+
+User-defined tags for Microsoft Azure was previously introduced as a Technology Preview in {product-title} 4.13 and is now generally available in {product-title} 4.14. For more information, see xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-user-defined-tags_installing-azure-customizations[Configuring the user-defined tags for Azure].
```
Comment (opayne1) [inline release_notes/ocp-4-14-release-notes.adoc:57]:
```suggestion
User-defined tags for Microsoft Azure was previously introduced as Technology Preview in {product-title} 4.13 and is now generally available in {product-title} 4.14. For more information, see xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-user-defined-tags_installing-azure-customizations[Configuring the user-defined tags for Azure].
```

**Review by opayne1 — COMMENTED** [review]
Just one small nit otherwise lgtm

**Pull request merged**